tool/cli.py

```diff
@@ -49,12 +52,37 @@ def create_pr(target_remote, target_branch):
     Fork selected repository
     """
     a = App()
-    s = a.guess_service(target_remote)
+    s = a.guess_service(remote=target_remote)
     s.create_pull_request(target_remote, target_branch, a.get_current_branch())
 
 
+@click.command(name="list-prs")
+@click.option('--service', "-s", type=click.STRING, default="github",
+              help="Name of the git service (e.g. github/gitlab).")
+@click.argument('repo', type=click.STRING, required=False)
+def list_prs(service, repo):
+    """
+    List pull requests of a selected repository, default to repo in $PWD
+    """
+    a = App()
+    if repo:
+        s = a.get_service(service, repo=repo)
+    else:
+        s = a.guess_service()
+    prs = s.list_pull_requests()
+    print(tabulate([
```

Comment:
The table looks great! I'm thinking only about output for the empty list. Since I have no open pull requests, I was a little confused on the first try. (If there is any problem or only no pull requests.)